xapian-bindings/python/doxy2swig.py

```diff
@@ -39,7 +40,7 @@ def my_open_write(dest):
     if hasattr(dest, "write"):
         return dest
     else:
-        return open(dest, 'w')
+        return io.open(dest, 'w', encoding = 'utf8')
 
 
 class Doxy2SWIG:
```

Comment:
Probably the final step before merging will be to rebase to squash the "fixup" commits onto the commits they fixup - I'd suggest we rebase onto git master as part of that, which should automatically drop this commit as already applied (and similarly should drop the trailing whitespace change that's in another commit).